IT_manager/settings.py

```diff
@@ -20,64 +20,80 @@
 # See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'django-insecure-i(tu#l!w7(*t5zn3mr&@wk#4j_=p&!k7^n1+1isqxy3xjgi-^8'
+SECRET_KEY = ("django-insecure-i(tu#l!w7"

```

Comment:
you must save all private information or in local setting or in env file

Reply:
> you must save all private information or in local setting or in env file
Yes, thank`s